server/src/controllers/v1/auth.js

```diff
@@ -0,0 +1,93 @@
+import User from '../../models/user';
+import { newToken } from '../../utils/helpers/jwtHelper';
+import errorMessage from '../../utils/errorMessage';
+import statusMessage from '../../utils/statusMessage';
+import destructureUser from '../../utils/destructureUser';
+
+const controllerStatus = (req, res) => res.status(200).json({ status: 'online' });
+
+const signinController = async (req, res) => {
+  let user;
```

Comment:
- make this a const
- move inside try, it has lexical scoping, so I don't see any issue with it, unless you have a reason to use it outside, I'm happy to discuss :) 
- if !user is shouldn't be thrown, that's expected behaviour, don't throw, return a response with message that account doesn't exist sign up
- if await throws , catch it in catch block, that should be intented behaviour of try/catch 

leme know if you have more doubts on this

apologoies for misspellings :D